src/robusta/integrations/slack/sender.py

```diff
@@ -411,6 +413,11 @@ def __create_finding_header(
         self, finding: Finding, status: FindingStatus, platform_enabled: bool, include_investigate_link: bool
     ) -> MarkdownBlock:
         title = finding.title.removeprefix("[RESOLVED] ")
+
+        match = MENTION_PATTERN.search(title)
+        mention = f" {match.group(0)}" if match else ""
+        title = MENTION_PATTERN.sub("", title).strip()
```

Comment:
Shouldn't this only be done if there is match?

Reply:
fixed